surveys/static/js/charts.js

```diff
@@ -199,6 +202,34 @@ ChartHelper.prototype._getCountChartData = function(dataSourceId, chartTitle) {
   return this._getChartData(dataSourceId, initChartDataFunc, castFunc, updateChartDataFunc, aggFunc);
 }
 
+ChartHelper.prototype._getDistributionChartData = function(dataSourceId, chartTitle) {
+  /**
+   * Get chart data for a chart of the "count" type.
+   * @param {String} dataSourceId - The ID of the primary data source to display.
+   * @param {String} chartTitle - The title of the chart to display.
+   */
+  function initChartDataFunc() { return []; }
+  var castFunc = String;
+  function updateChartDataFunc(chartData, savedData) {
+    var categoryFound = false;
+    // If an entry for this category exists in the chartData array, increment its counter
+    for (var i = 0; i < chartData.length; i++) {
+      var categoryName = chartData[i][0];
+      if (categoryName === savedData) {
+        chartData[i][1] += 1;
+        categoryFound = true;
+        break;
+      }
+    }
+    if (!categoryFound) {
+      // Insert an entry for this count in the chartData array
+      chartData.push([savedData, 1]);
+    }
+  }
+  var aggFunc = percentiles;
+  return this._getChartData(dataSourceId, initChartDataFunc, castFunc, updateChartDataFunc, aggFunc);
```

Comment:
What do you think about adding something here to order the savedData? I'd expect the x axis here to go in ascending order:

![Screen Shot 2019-07-03 at 2 38 14 PM](https://user-images.githubusercontent.com/1094243/60620248-50f3e480-9da0-11e9-8b21-e6abdd5ad30d.png)

Reply:
Great point! Added sorting in https://github.com/datamade/just-spaces/pull/192/commits/d48648461eb70197646edd9ea775c01c4c3bcf58.